zipline/finance/performance/tracker.py

```diff
@@ -77,7 +78,7 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, trading_calendar, env):
```

Comment:
Is it also ok to zero out the `excess_return` field as well? (The excess return of the algorithm over the treasuries)